client/src/components/Checkout/Checkout.tsx

```diff
@@ -0,0 +1,62 @@
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { FormikHelpers } from 'formik';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { checkout } from '../../helpers/APICalls/checkout';
+import CheckoutForm from './CheckoutForm/CheckoutForm';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+export default function Checkout(): JSX.Element {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+  const { loggedInUser } = useAuth();
+
+  const handleSubmit = (
+    { priceId, quantity }: { priceId: string; quantity: number },
+    { setSubmitting }: FormikHelpers<{ priceId: string; quantity: number }>,
+  ) => {
+    if (loggedInUser != undefined && loggedInUser) {
```

Comment:
The same comment as with testing for truthiness above. Just a nitpick!

Reply:
👍🏻Done

Reply:
I learn a lot from your comments😊